src/interpreter/executors/main.rs

```diff
@@ -193,7 +193,20 @@ where
 
             // TODO set tree balance
 
-            let program = self.run_program();
+            // `Interpreter` supports only `Create` and `Script` transactions. It is not `Create` ->
+            // it is `Script`.
+            let program = if !self
+                .transaction()
+                .as_script()
+                .expect("It should be `Script` transaction")
+                .script()
+                .is_empty()
+            {
+                self.run_program()
+            } else {
+                Ok(ProgramState::Skipped)
```

Comment:
While this is a better variant, I think we should just do `ProgramState::Return(1)` like create txs did before to avoid extra breaking changes downstream.

Reply:
Yeah, let's introduce `ProgramState::Skipped` together with an optimal solution.